common/sass/mixins.scss

```diff
@@ -69,3 +69,30 @@
     }
   }
 }
+
+@mixin triangle($size, $color, $direction) {
+  height: 0;
+  width: 0;
+  border-color: transparent;
+  border-style: solid;
+  border-width: $size / 2;
+
+  @if $direction == up {
+    border-bottom-color: $color;
+  } @else if $direction == right {
+    border-left-color: $color;
+  } @else if $direction == down {
+    border-top-color: $color;
+  } @else if $direction == left {
+    border-right-color: $color;
+  }
+}
+
+@mixin show-tooltip-on-hover {
+  &:hover .Tooltip {
+    opacity: 1;
+    visibility: visible;
+    transform: translate(-50%, -100%) translateY(-8px);
+    transition-delay: 400ms, 400ms, 300ms;
+  }
+}
```

Comment:
I felt that this was the lowest friction way to implement tooltips, instead of making the component take in a whole bunch of props for showing and hiding. You simply add this mixin to the parent element you want to show the tooltip on hover. Let me know if y'all would rather have it be component props. Maybe we can do either / or?